src/ansys/fluent/core/session_solver.py

```diff
@@ -107,3 +113,10 @@ def preferences(self):
         if self._preferences is None:
             self._preferences = _get_preferences(self)
         return self._preferences
+
+    @property
```

Comment:
@prmukherj in the Fluent solver's Python console, we have both `solverworkflow` _and_ `workflow`. `solverworkflow` is analogous to `meshing`, whcih just has the command implementations.